docs/modules/ROOT/pages/migration.adoc

```diff
@@ -1973,6 +1973,423 @@ Saml2AuthenticationRequestResolver authenticationRequestResolver() {
 Since Spring Security only supports the `POST` binding for authentication, there is not very much value in overriding the protocol binding at this point in time.
 ====
 
+[[use-new-requestmatchers]]
+==== Use the new `requestMatchers` methods
```

Comment:
Because this has four equals signs, I think it will appear under the SAML deprecations, which seems like an oversight.